project/Doc.scala

```diff
@@ -157,7 +157,6 @@ object Unidoc extends AutoPlugin {
 
   lazy val GenjavadocCompilerPlugin = config("genjavadocplugin") hide
 
-  override def trigger = allRequirements
```

Comment:
Stop adding this by default to all projects. Instead, it has been added to `RuntimeLibPlugins` instead, which is enabled by all of the modules that end users might actually depend on.